cupy/fft/fft.py

```diff
@@ -542,7 +544,8 @@ def fft(a, n=None, axis=-1, norm=None):
             is not given, the length of the input along the axis specified by
             ``axis`` is used.
         axis (int): Axis over which to compute the FFT.
-        norm (None or ``"ortho"``): Keyword to specify the normalization mode.
+        norm (None or ``"ortho"`` or ``"unnormalize"``): Keyword to specify
```

Comment:
Could you fix to take "forward" or "backward" as NumPy does?

Reply:
Hi all. I still plan to work on this, but got busy at work those days, I will do my best in the coming weeks.

Reply:
Any updates?